lib/committee/rails/test/methods.rb

```diff
@@ -3,24 +3,51 @@ module Test
     module Methods
       include Committee::Test::Methods
 
-      def schema_path
-        Rails.root.join('docs', 'schema', 'schema.json')
+      def committee_schema
+        @committee_schema ||= begin
+          driver = Committee::Drivers::HyperSchema.new
+          schema_hash = JSON.parse(File.read(Rails.root.join('docs', 'schema', 'schema.json')))
```

Comment:
Sorry, I'm getting confused. Here it assumes that docs/schema/schema.json exists, but the path in fake_app is schema/schema.json. It seems that test fails because of that, but I didn't change this path. Could you check this?